Formula/h/hdf5.rb

```diff
@@ -43,32 +42,18 @@ def install
               "settingsdir=$(libdir)",
               "settingsdir=#{pkgshare}"
 
-    system "autoreconf", "--force", "--install", "--verbose"
-
-    args = %W[
-      --disable-dependency-tracking
-      --disable-silent-rules
-      --enable-build-mode=production
-      --enable-fortran
-      --enable-cxx
-      --prefix=#{prefix}
-      --with-szlib=#{Formula["libaec"].opt_prefix}
-    ]
-    args << "--with-zlib=#{Formula["zlib"].opt_prefix}" if OS.linux?
-
-    system "./configure", *args
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
 
     # Avoid shims in settings file
     inreplace_files = %w[
-      src/H5build_settings.c
-      src/libhdf5.settings
-      src/Makefile
+      build/src/H5build_settings.c
+      build/src/libhdf5.settings
```

Comment:
Their CMake build configures those compiler wrappers separately, so we must replace in them as well.

```suggestion
      build/src/libhdf5.settings
      build/CMakeFiles/h5cc
      build/CMakeFiles/h5hlcc
```